lmdeploy/turbomind/turbomind.py

```diff
@@ -529,7 +529,6 @@ def end(self, session_id: int):
     async def async_end(self, session_id: int):
         """End the given session."""
         self.end(session_id)
-        await asyncio.sleep(0.002)
```

Comment:
what does `asyncio.sleep` result in?

Reply:
As described in https://github.com/InternLM/lmdeploy/issues/1788, the server can finish `async_cancel` after removing this line. Or else, `asyncio.sleep(0.002)` could not finish.

Reply:
When we introduce the `asyncio.sleep(0.002)`? Did we do that for some reason?

Reply:
> When we introduce the `asyncio.sleep(0.002)`? Did we do that for some reason?

To make it an async function. But there is no need to add this since it does not improve concurrency. There are no subsequent codes below the `sleep` for the CPU to run other codes and then come back.